README.md

```diff
@@ -0,0 +1,239 @@
+# swo-cli command-line search client for SolarWinds Observability log management service
```

Comment:
How about something like command-line client for SolarWinds Observability platform?

Reply:
changed to: `# swo-cli command-line client for SolarWinds Observability platform`